config/environments/test.rb

```diff
@@ -39,7 +39,4 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
-
-  config.neo4j.session_type = :server_db
-  config.neo4j.session_path = 'http://localhost:7475'
```

Comment:
How would you recommend setting this up? I was thinking to use Figaro and 2 env variables.